lib/test_prof/factory_bot.rb

```diff
@@ -1,13 +1,9 @@
 # frozen_string_literal: true
 
 module TestProf # :nodoc: all
-  FACTORY_GIRL_NAMES = {"factory_bot" => "::FactoryBot", "factory_girl" => "::FactoryGirl"}.freeze
```

Comment:
This commit https://github.com/test-prof/test-prof/commit/ce7a0417e2fbbc8687ef03a3614fd5abe6653dd2#diff-916f72dde19e12fa99062ee405a8c3335c938fcefcc7f8758851dca6c82f9c55 has introduced this file.